build_tools/upload_logs_to_s3.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import os
```

Comment:
Can you add a brief description that include some usage instructions?

Reply:
Added.